ngx_lua_sample/Makefile

```diff
@@ -0,0 +1,10 @@
+build:
+	docker-compose build nginx
+
+up: down
+	docker-compose up nginx
```

Comment:
```suggestion
	docker-compose up -d nginx
```
Faced this issue on Mac https://stackoverflow.com/questions/50121943/docker-compose-hangs-on-attaching-to

Reply:
My intention was to leave it attached in tab, so you can see the logs and force the open state :)! 

Vut I can change if you want.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/55913/150344123-358e0154-2883-4c7d-b719-3795e0a6cac1.png">

Reply:
like using multiple tabs so you can simulate, control your faulty but at the same time see the logs

Reply:
https://user-images.githubusercontent.com/55913/150345782-cfe22e55-ab54-4878-8048-b694e62ea07c.mp4

I recorded my screen, so you can see what I meant... maybe we should add this video to the readme as well?! I don't know.

Reply:
Oh right, it's helpful to have the logs, so lets leave it the way it is.

Reply:
And the video demonstration looks good too. Let's include it in the documentation.

Reply:
On a side note, I think I spot a bug in the code. The log statements which notify state changes are not correct, there seems to be an issue with the (order of the) arguments passed to the notify function. Looks like the first argument being passed is self. I'll check it out.

Reply:
On this line, https://github.com/dream11/lua-circuit-breaker/blob/master/src/breaker.lua#L134, the notify function should be called like this
`self.notify(cb_name, new_state)`
Also, there should be a check if notify is nil or not since there is no default value. I'll fix this issue.